doc/Type/Metamodel/MethodDelegation.pod6

```diff
@@ -0,0 +1,67 @@
+=begin pod :kind<Type> :subkind<class> :category<metamodel>
+
```

Comment:
I guess this would rather be `subkind<role>`

Reply:
That's correct...